src/python/Utils/ExtendedUnitTestCase.py

```diff
@@ -23,7 +24,7 @@ def assertContentsEqual(self, expected_obj, actual_obj, msg=None):
         """
 
         def traverse_dict(dictionary):
-            for key, value in dictionary.iteritems():
+            for key, value in iteritems(dictionary):
```

Comment:
Dario, this documentation:
https://python-future.org/_modules/future/utils.html

seems to suggest to only use `iteritems` if we have python< 2.7. Perhaps we should instead use `viewitems, viewkeys, viewvalues`?